frontend/packages/dev-console/src/components/pipelineruns/logs/MultiStreamLogs.tsx

```diff
@@ -146,16 +146,19 @@ export const MultiStreamLogs: React.FC<MultiStreamLogsProps> = ({
           onScroll={handleScrollCallback}
         >
           {containers.map((container, idx) => {
+            const resourceStatus = containerToLogSourceStatus(containerStatus[idx]);
             return (
-              <Logs
-                key={container.name}
-                resource={resource}
-                container={container}
-                resourceStatus={containerToLogSourceStatus(containerStatus[idx])}
-                onComplete={handleComplete}
-                render={renderToCount >= idx}
-                autoScroll={autoScroll}
-              />
+              resourceStatus !== LOG_SOURCE_NOT_READY && (
```

Comment:
@sahil143  Do not render the log component until it is in ready/running state. This will fix the issue.
```suggestion
           ![LOG_SOURCE_WAITING, LOG_SOURCE_NOT_READY].includes(resourceStatus) && (
```

Reply:
@karthikjeeyar I already tried this solution, This works but for some pipelines which have one or two line logs this doesn't work. because the next task is shown before any logs can be printed out
![mock-app-logs](https://user-images.githubusercontent.com/9278015/83108619-b00bc780-a0dd-11ea-8813-d2ba7df0b83b.gif)